pyproject.toml

```diff
@@ -16,7 +16,7 @@ readme = "README.md"
 keywords = ["database", "nosql", "vector", "search"]
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<4.0"
+python = ">=3.7,<4.0"
```

Comment:
flake8 was asking for `>=3.8.1` dependency

Reply:
I have seen major libraries supporting 3.7, the security support is about to expire though. There could be users still on 3.7 trying out Tigris. 

https://endoflife.date/python

Reply:
Ok, let's keep it at 3.7